docs/examples/views/requirements.txt

```diff
@@ -0,0 +1,6 @@
+Deprecated==1.2.13
```

Comment:
Nit: Where is the need for `deprecated`, `wrapt` and `typing_extensions`?

Reply:
This is from pip freeze output after installing the `opentelemetry-sdk==1.12.0rc2`.